Bindings/Java/Matlab/Utilities/osimC3Dconverter.m

```diff
@@ -174,61 +175,40 @@ function osimC3Dconverter(varargin)
             end
         end
     end
-
-    %% change the order of the forces
-    % turn the timetable into a Maltab Struct
-    sdata = osimTableToStruct(forces_flattened);
-    % get the label names
-    labels =  fieldnames(sdata);
-    % find all the labels with string m (including time)
-    imoments = find(~cellfun(@isempty,strfind(labels,'m')));
-    % get a tempory set of labels for the moements, delete them from the
-    % original
-    templabels = labels(imoments);
-    labels(imoments) = [];
-    % append the moment labels
-    labels2 = [labels;templabels];
-    % reorder the structure given the new label order
-    sdata = orderfields(sdata, labels2);
-
+    
     %% Check for NaNs
     % If there are any NaNs in the forces data, OpenSim will interpret them as
     % the 'end' of the file. We must set NaNs to zero and report where they are
```

Comment:
Is this true that we cannot read in NaNs correctly? How does the marker reader/writer work then? We should not have to change NaNs to 0 and this is in fact incorrect to do. Please send me a .mot with some  NaNs to confirm this issue. This should be fixed on the API side.